### PR TITLE
Core/DBC Move WorldSafeLocs.dbc storage to DB

### PR DESCRIPTION
**Pull request by Winfidonarleyan**
**Changes proposed:**
-  Moved storage from WorldSafeLocs.dbc to DB
- Added new command **.reload game_graveyard**

**Target branch(es):** master

**Issues addressed:** https://github.com/azerothcore/azerothcore-wotlk/issues/584

**Tests performed:** Tested in game, tested in build. (Win 10 / VS 15 2017)

**How to test the changes:**
1. Apply commit
2. Rebuild Cmake
3. Rebuild Project
4. Try to die :)

**TODO list:**

- [x] Try to die in open world
- [x] Try to die in battlegroud
- [x] Try to die in instance

### CODE REVIEW

**Comment by BarbzYHOOL** [comment]
Nice!!

File: src/server/game/Misc/GameGraveyard.cpp [hunk 1]
```diff
@@ -0,0 +1,371 @@
...
+
+    return true;
+}
+
+void Graveyard::RemoveGraveyardLink(uint32 id, uint32 zoneId, TeamId teamId, bool persist /*= false*/)
+{
+    GraveyardMapBoundsNonConst range = GraveyardStore.equal_range(zoneId);
+    if (range.first == range.second)
+    {
+        sLog->outError("Table `game_graveyard_zone` incomplete: Zone %u Team %u does not have a linked graveyard.", zoneId);
```
Comment (Nefertumm) [inline src/server/game/Misc/GameGraveyard.cpp]: Add teamId here.

**Comment by Winfidonarleyan** [comment]
* WorldSafeLocs.dbc - this game graveyard
* Not used by the client
> the name of the graveyards?

In what sense?

**Comment by BarbzYHOOL** [comment]
"Not used by the client"

then why is it in the dbc xD i'm just curious because if we modify this without updating the dbc client side, we might miss something (in THEORY)

**Comment by FrancescoBorzi** [comment]
is this ready to be tested?

**Comment by Winfidonarleyan** [comment]
Yes

**Comment by wetbrownsauce** [comment]
Tested a bit

- Tested new .reload game_graveyard by changing the z value of a graveyard to high up in the air, reloading with command, and then dying at that graveyard. Releasing correctly put me at the changed value. Command and reload work. 

- Died in a few zones in Kalimdor, Eastern Kingdoms, Outlands, Northrend. All took me to nearby graveyards as expected. 

- Died in a couple dungeons in each of the above areas, all took me to nearby graveyards.

- Died in every battleground, all battleground graveyards seem functional. 

I only tested on horde side, but no issues so far.

**Comment by BarbzYHOOL** [comment]
Again, I'm asking if the DBC is used by the client for something (visually on the minimap for example)

DBC = database client if I recall

https://wowdev.wiki/DB/WorldSafeLocs

What happens if you remove the dbc client side too? no names on the graveyards?

**Comment by BarbzYHOOL** [comment]
Can someone try to remove that dbc serverside and client side before we merge? just for curiosity lol

**Comment by BarbzYHOOL** [comment]
@Winfidonarleyan what's this last commit? it's from another PR no?

**Comment by FrancescoBorzi** [comment]
can we merge this one?

**Comment by Winfidonarleyan** [comment]
I think yes

**Pull request merged**